modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -50,14 +50,17 @@ You can enclose a range of characters in square brackets, to match against all o
 | Matches any alphanumeric character.
 
 | `^`
-| Matches beginning of input.
-If the multi-line flag is set to `true`, also matches immediately after a line-break character.
-For example, `/^A/` does not match the '[.code]``A``' in "[.code]``an A``", but does match the '[.code]``A``' in "[.code]``An E``".
+| Matches the beginning of input.
+For example, `/^a/` matches `abb` and `ab`; but does _not_ match `ba`, `bab`, or `bba`.
```

Comment:
Not the focus of the PR, but surrounding the regex string with slashes seems odd here. It's a convention for "this is a regex", but *not* something you do when you enter it in the product, so might not make sense to add them here?

Reply:
(I do really like the change of examples here btw!)

Reply:
Okay, I have removed the slashes.